releasenotes/notes/deprecate-span-meta-ad1c96ba55acaac9.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    ``Span.meta`` has been deprecated. Use ``Span.get_tag(key)`` and ``Span.set_tag(key, val)`` instead.
```

Comment:
Can we update this to add references in the release notes to the public api methods to use instead?